### PR TITLE
Refactor: moderation

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- updates moderation items lists after moderation
- fixes layout
- fixes broken avatar src

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/hooks/src/index.ts [hunk 1]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
 export {
   constants,
   useModeration,
+  useModerationStatus,
+  useCheckModerator,
+  useGetCount,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: naming is too generic, what if I also name a hook in `use-trending` as `useGetCount` ?

File: ui/hooks/src/index.ts [hunk 2]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
...
+  useGetFlags,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: same

File: ui/hooks/src/index.ts [hunk 3]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
...
+  useInfiniteLog,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: same as above

File: ui/hooks/src/index.ts [hunk 4]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
...
+  useInfinitePending,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: same as above

File: ui/hooks/src/index.ts [hunk 5]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
...
+  useInfiniteKept,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: same as above

File: ui/hooks/src/index.ts [hunk 6]
```diff
@@ -11,6 +21,14 @@ import useReasons from './use-reasons';
...
+  useInfiniteDelisted,
```
Comment (SeverS) [inline ui/hooks/src/index.ts]: same as above

File: ui/design/src/components/ErrorLoader/styled-elements.ts [hunk 7]
```diff
@@ -24,14 +24,15 @@ const PageWrapper: any = styled(Box)`
 `;
 
 const ContentWrapper: any = styled(Box)`
-  margin: auto;
-  position: absolute;
-  top: 40%;
+  margin: 15rem auto auto;
```
Comment (SeverS) [inline ui/design/src/components/ErrorLoader/styled-elements.ts]: are you sure it's working on mobile? 15rem @16px is about 240px

File: ui/design/src/components/ErrorLoader/styled-elements.ts [hunk 8]
```diff
@@ -24,14 +24,15 @@ const PageWrapper: any = styled(Box)`
...
   transform: translate(0, -50%);
```
Comment (SeverS) [inline ui/design/src/components/ErrorLoader/styled-elements.ts]: you don't need this

File: apps/moderation/src/components/prompt-authentication.tsx [hunk 9]
```diff
@@ -22,7 +22,7 @@ const PromptAuthentication: React.FC<IPromptAuthenticationProps> = props => {
 
   return (
     <ModerationAppErrorCard
-      size={'18.75rem'}
+      boxSize={'18.75rem'}
```
Comment (quininez) [inline apps/moderation/src/components/prompt-authentication.tsx:25]: does this scale on mobile?

File: apps/moderation/src/components/prompt-authorization.tsx [hunk 10]
```diff
@@ -12,7 +12,7 @@ const PromptAuthorization: React.FC<IPromptAuthorizationProps> = props => {
   const { titleLabel, subtitleLabel } = props;
   return (
     <ModerationAppErrorCard
-      size={'14rem'}
+      boxSize={'14rem'}
```
Comment (quininez) [inline apps/moderation/src/components/prompt-authorization.tsx:15]: same here, is it also on mobile?

File: apps/moderation/src/components/content-card/content.tsx [hunk 11]
```diff
@@ -98,7 +102,11 @@ const Content: React.FC<IContentProps> = props => {
             <Text margin={{ right: '0.2rem' }}>{props.originallyReportedByLabel}</Text>
             <Avatar
               ethAddress={props.reporter || ''}
-              src={props.reporterAvatar}
+              src={
+                props.reporterAvatar
+                  ? `${ipfsGateway}/${props.reporterAvatar}`
```
Comment (quininez) [inline apps/moderation/src/components/content-card/content.tsx:107]: why was this removed from the hook and added here?

Comment (SeverS) [reply apps/moderation/src/components/content-card/content.tsx:107]: we had a weird behaviour when using it in the hook and the hook refetches, this is from the moderation api response not the entry fetch response

File: apps/moderation/src/components/prompt-authentication.tsx (hunk 9 above)
Comment (josenriagu) [reply apps/moderation/src/components/prompt-authentication.tsx:25]: yeap

Comment (josenriagu) [reply apps/moderation/src/components/prompt-authentication.tsx:25]: I only renamed the prop

**Review by SeverS — APPROVED** [review]
